egs/swbd/s5c/local/nnet3/run_tdnn.sh

```diff
@@ -36,6 +38,35 @@ suffix=
 if [ "$speed_perturb" == "true" ]; then
   suffix=_sp
 fi
+
+cnn_opts=()
+if [ "$use_cnn" == "true" ]; then
+# When the cnn option is on, the LDA is replaced by
+# an CNN layer at the front of the network
+# and the recipe uses fbank features as the CNN input
+# As the dimension of the CNN output is usually large, we place
+# a linear layer at the output of CNN for dimension reduction
+# The ivectors are processed through a fully connected affine layer,
+# then concatenated with the CNN bottleneck output and
+# passed to the deeper part of the network.
+# Due to the data compression issue, it is better to convert MFCC
+# to FBANK in the network instead of directly using FBANK features
+# from the storage. This script uses MFCC features as its input
+# and were converted to FBANK features with an inverse of DCT matrix
+# at the first layer of the network.
+
+  cnn_layer="--filt-x-dim=3 --filt-y-dim=8 --filt-x-step=1 --filt-y-step=1 --num-filters=1024 --pool-x-size=1 --pool-y-size=3 --pool-z-size=1 --pool-x-step=1 --pool-y-step=3 --pool-z-step=1"
```

Comment:
are you using 1024 filters in the recipe from which you generated the above results ?